vips/image.go

```diff
@@ -170,12 +169,28 @@ func (r *ImageRef) GetOrientation() int {
 	return vipsGetMetaOrientation(r.image)
 }
 
-func (r *ImageRef) SetOrientation(orientation int) {
-	vipsSetMetaOrientation(r.image, orientation)
+func (r *ImageRef) SetOrientation(orientation int) error {
+	out, err := vipsCopyImage(r.image)
```

Comment:
we need to understand the penalty of this copy operation. is it a copy-on-write ?

Reply:
https://libvips.github.io/libvips/API/current/libvips-conversion.html#vips-copy

> VIPS copies images by copying pointers, so this operation is instant, even for very large images.

https://github.com/lovell/sharp/commit/bb15cd906717ba054d9c23db2971e94d2e250969

Reply:
cool. using a pointer